Setup.bat

```diff
@@ -17,13 +17,11 @@ call :MarkStartOfBlock "Setup the git hooks"
     rem Remove the old post-merge hook.
     if exist .git\hooks\post-merge del .git\hooks\post-merge
 
-    rem Add git hook to run Setup.bat when RequireSetup file has been updated.
-    echo #!/usr/bin/env bash>.git\hooks\post-merge
-    echo changed_files="$(git diff-tree -r --name-only --no-commit-id ORIG_HEAD HEAD)">>.git\hooks\post-merge
-    echo check_run() {>>.git\hooks\post-merge
-    echo echo "$changed_files" ^| grep --quiet "$1" ^&^& exec $2>>.git\hooks\post-merge
-    echo }>>.git\hooks\post-merge
-    echo check_run RequireSetup "cmd.exe /c Setup.bat %*">>.git\hooks\post-merge
+    rem Remove the old pre-commit hook.
+    if exist .git\hooks\pre-commit del .git\hooks\pre-commit
+
+    rem Copy git hooks to .git directory.
+    xcopy /s /i /q "%~dp0\SpatialGDK\Extras\git" "%~dp0\.git\hooks"

```

Comment:
Previously, if you ran `Setup.bat` with arguments (e.g. `--mobile` or `--china`), these would be saved into the git hook. Could you add a placeholder into the hook inside `SpatialGDK\Extras\git` and replace it when copying it to `.git`?